core/src/main/java/org/openstack4j/core/transport/ObjectMapperSingleton.java

```diff
@@ -38,7 +38,9 @@ private ObjectMapperSingleton() {
     
 
     public static ObjectMapper getContext(Class<?> type) {
-        return type.getAnnotation(JsonRootName.class) == null ? INSTANCE.mapper : INSTANCE.rootMapper;
+        return type.getAnnotation(JsonRootName.class) == null ?
+                type.getAnnotation(SerializationAllowNull.class) == null ? INSTANCE.mapper : INSTANCE.mapper.setSerializationInclusion(Include.ALWAYS) :
+                type.getAnnotation(SerializationAllowNull.class) == null ? INSTANCE.rootMapper : INSTANCE.rootMapper.setSerializationInclusion(Include.ALWAYS);
```

Comment:
Careful here, it changes the state of the shared mappers so it will propagate to unrelated serializations.